cli/.eslintrc.cjs

```diff
@@ -27,6 +27,11 @@ module.exports = {
     }
   },
   rules: {
+    quotes: [
+      'error',
```

Comment:
I don't think `error` should be thrown around like that.

Things that can potentially lead to serious or breaking outcomes should be set to the error level, but otherwise should be set at a warning level.

Reply:
Hi @jayantbh,
I gave changed to warn , the linter will still detect violations (like using double quotes), but instead of failing, it will issue a warning